controller/jobs.go

```diff
@@ -144,9 +146,39 @@ func (c clusterClientWrapper) Hosts() ([]utils.HostClient, error) {
 	return res, nil
 }
 
-type clusterClient interface {
-	Host(string) (utils.HostClient, error)
-	Hosts() ([]utils.HostClient, error)
+func (c clusterClientWrapper) StreamHostEvents(ch chan *discoverd.Event) (stream.Stream, error) {
+	hostChan := make(chan *discoverd.Event)
+	stream, err := c.Client.StreamHostEvents(hostChan)
```

Comment:
Oh yes. When this was returning `utils.HostClient`s, then this wrapper was necessary. It might not be necessary any more.